src/Scene.ts

```diff
@@ -682,18 +683,114 @@ export class Scene {
     this.COMvisual.add(mesh);
   }
 
-  public addSky(): void {
-    var cubeLoader = new THREE.CubeTextureLoader();
-    var cubeTexture = cubeLoader.load([
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-negx.jpg',
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-posx.jpg',
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-posy.jpg',
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-negy.jpg',
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-negz.jpg',
-      'https://fuel.gazebosim.org/1.0/openrobotics/models/skybox/tip/files/materials/textures/skybox-posz.jpg',
-    ]);
-
-    this.scene.background = cubeTexture;
+  public addSky(cubemap: string | undefined): void {
+    if (cubemap === undefined) {
+      var cubeLoader = new THREE.CubeTextureLoader();
```

Comment:
```suggestion
      const cubeLoader = new THREE.CubeTextureLoader();
```

Reply:
If we expect to load an asset via websocket, we should use the websocket loading manager in loaders.

I believe this is not the case (we get it from fuel), but it's still worth mentioning.